### ISSUE TITLE
ESLint fails on all my projects (working on VSCode)

### ISSUE THREAD

**Issue by destegabry** [apexskier/nova-eslint#279]
ESLint doesn't work at all. All the projects I tested is working out of the box on VSCode with ESLint extension.

**Environment**

- Nova version: `9.1 build 383286`
- Extension version: `1.7.2`
- ESLint version: `8.11.0`
- macOS version: `12.2.1`
- NodeJS information:
  - node version: `17.7.1`
  - npm version: `8.5.2`
  - installation method: homebrew

**Extension console output**

```sh
ESLint[9:38:48.428000] Failed to lint

ESLint[9:38:48.428000]   stderr:  
Oops! Something went wrong! :(

ESLint: 8.11.0

No files matching the pattern "/Code/{client-name}/{project-name}/webapp/src/App.tsx" were found.
Please check for typing mistakes in the pattern.



ESLint[9:38:48.428000]   command:  /Users/gabriele/Code/{client-name}/{project-name}/webapp/node_modules/.bin/eslint

ESLint[9:38:48.428000]   args:  /Code/{client-name}/{project-name}/webapp/src/App.tsx --fix --format=json

ESLint[9:38:48.429000] Extension encountered an uncaught exception:
/Users/gabriele/Library/Application Support/Nova/Extensions/apexskier.eslint/Scripts/main.dist.js (Line 372, Column 32)
Error: failed to lint (2)
    @file:///Users/gabriele/Library/Application%20Support/Nova/Extensions/apexskier.eslint/Scripts/main.dist.js:372:32
    @[native code]
    promiseReactionJob@[native code]

```

**Additional context**

I opened the folder `/Users/gabriele/Code/{client-name}/{project-name}/webapp` in Nova.

I'm a bit worried about this line:

```
No files matching the pattern "/Code/{client-name}/{project-name}/webapp/src/App.tsx" were found.
```

Because that seems an absolute path and it shouldn't. The correct path is `/Users/gabriele/Code/{client-name}/{project-name}/webappsrc/App.tsx`

**Comment by jbrowning** [comment]
Same thing here. 

In my case my project is located at `/Users/username/Projects/project` but this extension is attempting to lint files at `/Projects/project`.

**Comment by apexskier** [comment]
Possibly related to #277

**Comment by timfee** [comment]
+1
```
ESLint[15:26:02.387000]   stderr:  
Oops! Something went wrong! :(

ESLint: 8.12.0

No files matching the pattern "/Sites/kr/pages/index.tsx" were found.
Please check for typing mistakes in the pattern.



ESLint[15:26:02.387000]   command:  /Users/timfee/Sites/kr/node_modules/.bin/eslint

ESLint[15:26:02.387000]   args:  /Sites/kr/pages/index.tsx --fix --format=json
```

**Comment by TheJoffrey** [comment]
Still same case. Does anyone find a way to use ESLint in Nova?

**Comment by apexskier** [comment]
Resolved by https://github.com/apexskier/nova-eslint/pull/283

**Comment by apexskier** [comment]
Included in release [v1.7.3](https://github.com/apexskier/nova-eslint/releases/tag/v1.7.3)
